bin/xbps-uhelper/main.c

```diff
@@ -64,20 +64,10 @@ usage(void)
 	"    real-version\t<pkgname> ...\n"
 	"\n"
 	"  Options shared by all actions:\n"
-	"    -C\t\tPath to xbps.conf file.\n"
-	"    -d\t\tDebugging messages to stderr.\n"
-	"    -r\t\t<rootdir>\n"
-	"    -V\t\tPrints the xbps release version\n"
-	"\n"
-	"  Examples:\n"
-	"    $ xbps-uhelper cmpver 'foo-1.0_1' 'foo-2.1_1'\n"
-	"    $ xbps-uhelper getpkgdepname 'foo>=0'\n"
-	"    $ xbps-uhelper getpkgdepversion 'foo>=0'\n"
-	"    $ xbps-uhelper getpkgname foo-2.0_1\n"
-	"    $ xbps-uhelper getpkgrevision foo-2.0_1\n"
-	"    $ xbps-uhelper getpkgversion foo-2.0_1\n"
-	"    $ xbps-uhelper pkgmatch foo-1.0_1 'foo>=1.0'\n"
-	"    $ xbps-uhelper version pkgname\n");
+	"    -C, --config\tPath to xbps.conf file.\n"
```

Comment:
```
 -C --config <dir>          Path to confdir (xbps.d)
```

I think indention should be done with spaces instead of `\t` between words since this is variable in shells.